src/env.h

```diff
@@ -240,6 +240,7 @@ namespace node {
   V(domains_stack_array, v8::Array)                                           \
   V(fs_stats_constructor_function, v8::Function)                              \
   V(generic_internal_field_template, v8::ObjectTemplate)                      \
+  V(inspector_console_api_object, v8::Object)                                 \
```

Comment:
As @Fishrock123 said, this is just used as a bucket of named persistent values from native code, it's never exposed to JS.